connect/runtime/src/main/java/org/apache/kafka/connect/runtime/errors/DeadLetterQueueReporter.java

```diff
@@ -111,6 +116,7 @@ public static DeadLetterQueueReporter createAndSetup(Map<String, Object> adminPr
         this.connConfig = connConfig;
         this.connectorTaskId = id;
         this.errorHandlingMetrics = errorHandlingMetrics;
+        this.dlqTopicName = connConfig.dlqTopicName();
```

Comment:
Should we trim this?
```suggestion
        this.dlqTopicName = connConfig.dlqTopicName().trim();
```